pkg/apis/v1alpha1/suite_test.go

```diff
@@ -16,18 +16,23 @@ package v1alpha1_test
 
 import (
```

Comment:
Really nice test coverage! I feel like this covers a comprehensive set of cases!